README.md

```diff
@@ -53,7 +53,7 @@ ES6:
 import { ContentClient } from 'dc-delivery-sdk-js';
 
 const client = new ContentClient({
-  account: 'myaccount'
+  // configuration options,  see below...
```

Comment:
The reason for the removal is to ensure users correctly configure their client.

I thought about changing it to `hubName`, but CD2 isn't provisioned for everyone and so this might raise provisioning questions. 

These examples are demonstrating how to use the SDK using ES6, CommonJS or via the prebundled js files and I think that's still clear.

Reply:
Changed it to `hubName`